llpc/test/shaderdb/PipelineTess_TestInOutPacking.pipe

```diff
@@ -0,0 +1,172 @@
+; BEGIN_SHADERTEST
+; RUN: amdllpc -spvgen-dir=%spvgendir% -v %gfxip %s | FileCheck -check-prefix=SHADERTEST %s
+; SHADERTEST-LABEL:.AMDGPU.disasm
+; SHADERTEST-LABEL:_amdgpu_ls_main
+; SHADERTEST: ds_write2_b32 v0, v6, v7 offset0:2 offset1:3
+; SHADERTEST: ds_write2_b32 v0, v6, v7 offset1:1
+; SHADERTEST-LABEL:_amdgpu_hs_main
+; SHADERTEST: ds_read2_b32 v[0:1], v2 offset0:2 offset1:3
+; SHADERTEST: ds_read2_b32 v[2:3], v2 offset1:1
+; SHADERTEST-LABEL:_amdgpu_vs_main
+; SHADERTEST: exp param0 v6, v7, v2, v3
+; SHADERTEST: exp param1 v8, v11, off, off
+; SHADERTEST: AMDLLPC SUCCESS
+; END_SHADERTEST
+
+[Version]
+version = 40
+
+[VsGlsl]
```

Comment:
it is better to add test with following stages
- vs + tcs + tes
- vs + tcs + tes + gs + fs

and please make sure in/out in vs->tcs  are packed.

Reply:
Okay. I will add two tests for it.